### PR TITLE
Updated example

### PR DESCRIPTION
**Pull request by Bossa573**

### CODE REVIEW

**Comment by Bossa573** [comment]
@fedor

File: README.md [hunk 1]
```diff
@@ -16,18 +16,30 @@ npm install tmfy
...
+		console.log('email sent');
+	})
+	.catch(function(error) {
+		// never called
+	});
+
+lib.sendEmailSlowlyTimeout(1000, 'me@apple.com') // timeout after 1000ms
 	.then(function(result) {
-		console.log('sleep first');
+		// never called
```
Comment (fedor) [inline README.md:42]: @Bossa573 added a comment

File: README.md [hunk 2]
```diff
@@ -16,18 +16,30 @@ npm install tmfy
...
+
 tmfy.timeifyAll(lib);
 
-lib.sleep_sTimeout(1000, 2) // timeout after 1000ms, sleep for 2s
+lib.sendEmailTimeout(1000, 'me@apple.com') // timeout after 1000ms
+	.then(function(result) {
+		console.log('email sent');
+	})
+	.catch(function(error) {
+		// never called
```
Comment (fedor) [inline README.md:37]: @Bossa573 added a comment

**Comment by BossaGroove** [comment]
cool

**Comment by fedor** [comment]
@Bossa573 added 2 comments to improve readability. Can you please take end result and update `sample.js`?

**Comment by BossaGroove** [comment]
I think the changes is already applied to the PR?

**Comment by BossaGroove** [comment]
oh wait, it's `sample.js` not `readme`

**Pull request merged**